src/js/redux/reducers/aboutTheData/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * aboutTheDataReducer.js
+ * Created by Max Kendall 12/8/2020
+*/
+
+export const initialState = {
+    details: [],
+    dates: [],
```

Comment:
I am assuming dates is the periods response? This also should be more descriptive.

Reply:
This is one i disagree w/ b/c throughout the code I refer to the table as the `dates` table. This is already nested inside `aboutTheData`. I can change in a follow up clean up pr at some point if necessary.